WordPress/gutenberg
# Dashicons: Remove non-existent icons from type

## What?

I noticed that there were two types of dashicons included that didn't have CSS defined, so I'm going to remove them.

https://github.com/t-hamano/wordpress-develop/blob/a27e6a8ecd5279a45e47ef88d048643740d07b5a/src/wp-includes/css/dashicons.css

## Why?

Because these two icon strings don't display anything.

## Testing Instructions

- Run `npm run build:package-types`
- In the icon prop below, enter the string this PR is trying to remove:
  https://github.com/WordPress/gutenberg/blob/b4c614f1268b6a5fb696c2b197bb8240ae6f1fc1/packages/block-editor/src/components/block-variation-picker/index.js#L26-L27
- Lists like the one below should not be displayed in this PR:
  ![image](https://github.com/user-attachments/assets/050541bb-2a1a-4f68-adb9-b3992bc6b269)
  ![image](https://github.com/user-attachments/assets/2e354ca1-6d98-4dc7-87af-5f3f09b39ddb)
